src/config/pci-passthrough.md

```diff
@@ -0,0 +1,405 @@
+# PCI Passthrough
+
+PCI pass-through allows you to pass through an extra graphics card in your
+system with little to no performance drawback. This allows you to run
+applications that require GPU acceleration (CAD programs, video games, etc.) in
+your virtual machine on operating systems like Windows 10 with almost bare metal
+performance.
+
+PCI pass-through requires a lot of trial and error. I recommend you read the
+external resources and read each section from beginning to end.
+
+## System Requirements
+
+The archwiki maintains a list of [example
+rigs](https://wiki.archlinux.org/index.php/PCI_passthrough_via_OVMF/Examples)
+for PCI pass-through. For general requirements:
+
+- Hardware must support hardware virtualization and IOMMU
+   - For Intel CPU: We require Intel VT-x and Intel VT-d support. You can check
+      if your CPU has this feature on [Intel
+      ark](https://ark.intel.com/content/www/us/en/ark/search/featurefilter.html?productType=873&0_VTD=True).
+   - All AMD CPUs: All CPUs after bulldozer are supported. Exceptions include:
+      CPUs from K10 generation and require a motherboard with
+      [890FX](https://www.amd.com/system/files/TechDocs/43403.pdf#page=18) or
+      [990FX](https://www.amd.com/system/files/TechDocs/48691.pdf#page=21).
+- Your motherboard must support IOMMU
+- Your Guest GPU ROM must support UEFI
+   - Check if your GPU has any ROM on this list: [Video BIOS
+      Collection](https://www.techpowerup.com/vgabios/). Most GPUs after 2012
+      support this.
+
+## Enabling IOMMU
+
+### BIOS
+
+First, enable IOMMU in your computer's BIOS. This setting is usually named
+"SVM", "IOMMU", "VT-d", "AMD-vi", or "virtualization technology". The help for
+your BIOS may or may not say whether that is the specific setting to enable
+IOMMU support. Some of the [example
+rigs](https://wiki.archlinux.org/index.php/PCI_passthrough_via_OVMF/Examples)
+may explain which settings they enabled in BIOS. You may have to resort to trial
+and error.
+
+### Kernel Parameter
+
+Add a kernel parameter to enable IOMMU for your CPU. For grub, edit
+`/etc/default/grub` and add one of the following to
+`GRUB_CMDLINE_LINUX_DEFAULT`:
+
+- Intel CPU: `intel_iommu=on`
+- AMD CPU: `amd_iommu=on`
+
+You should also add the `iommu=pt` kernel parameter to make sure the Linux
+doesn't touch the devices you have added to your pass-through.
+
+Once you have added your required kernel parameters, run the following command
+to generate `grub.cfg` with your new kernel parameters:
+
+```
+# grub-mkconfig /boot/grub/grub.cfg
+```
+
+### Testing
+
+The BIOS settings are very difficult to get right there is no standardized
```

Comment:
Suggest "The BIOS settings can be very difficult to get right, as there is no ..."